app.vue

```
@@ -110,16 +106,28 @@ const { data: xForwardedhost } = await useFetch('/api/getXForwardedHost');
 const flagStore = useFlagsStore();
 
-const fetchFlags = async() => {
+const { VS_AWS_APPCONFIG_URL } = useRuntimeConfig();
+
+// only get feature flag values when running on server
+if(process.server) {
     try {
-        const flags = featureFlagsData;
-        flagStore.flags = flags;
+        axios.get(VS_AWS_APPCONFIG_URL)
+        .then(function (response) {
```

ESLint

[error] Unexpected space before function parentheses.

(space-before-function-paren)